lib/graphql_connector/http_client.rb

```diff
@@ -34,6 +35,13 @@ def raw_query(query_string, variables: {})
 
     private
 
+    def handle_headers
+      return @headers if @connector.empty?
+
+      @headers
+        .merge(@connector[:base].send(@connector[:method]))
```

Comment:
**Optional**: Do we consider that method can be `private`? Otherwise we could also use `public_send`